docs/visaodoproduto/tap.md

```diff
@@ -0,0 +1,149 @@
+- Rejeição pelos órgãos públicos por inconformidade do relatório
+- Enviar relatórios para órgãos não responsáveis pela denúncia em questão
+- Usabilidade baixa e burocratização da aplicação
+
+O Scrum Master será o responsável por monitorar os riscos internos, e facilitar a adoção de medidas que mitiguem esses riscos, assim como o Product Owner será responsável por gerenciar os riscos externos. O controle é feito através das definições previstas na seção Gerenciamento de Riscos que pode ser encontrada [aqui](../metodologia/gerenciaRisco.md).
+
+## Cronograma e marcos
+|Marco|Data|Atividade|
+|--|--|--|
+|Início do projeto|06/02/21|Início da primeira Sprint do projeto|
+|Release 01|21/03/21 - 27/03/21|Primeira entrega de valor do projeto, que aborda os principais artefatos do início de projeto|
+|Release 02|16/05/21 - 22/05/21|Segunda entrega de valor, que aborda as funcionalidades definidas no escopo|
+
+## Custo Estimado
+Os custos estimados são para 15 sprints de projeto, ou seja, 15 semanas, totalizando 3 meses e 3 semanas de projeto.
+
+### Pessoal
+Utilizando como referência o site de busca de empregos **Glassdoor**, verificamos o salário de cada um dos papéis desempenhados pelo time.
+
+|Papel na equipe|Quantidade de membros na função|Transporte e alimentação/dia|Salário/mês|Total|
+|--|--|--|--|--|
+|Desenvolvedor|6|R$0,00|R$2.772,00|R$16.632,00|
+|DevOps|1|R$0,00|R$11.000,00|R$11.000,00|
+|Product Owner|1|R$0,00|R$7.549,00|R$7.549,00|
+|Scrum Master|1|R$0,00|R$7.945,00|R$7.945,00|
+|Arquiteto|1|R$0,00|R$10.000,00|R$10.000,00|
+|Total|||||R$53.126,00|
+
+Observações:
+- Os valores pagos em transporte e alimentação são nulos porque o trabalho é desenvolvido remotamente de forma integral.
+
+### Aquisições
+Para o levantamento de custos foram considerados computadores com a especificação de um processador Intel Core i7, 1 TB de Espaço de Armazenamento, 8 GB de memória RAM e Windows 10, todos os notebooks da marca Lenovo com o [custo até o dia 25/02/2021](https://www.zoom.com.br/notebook/notebook-lenovo-ideapad-s145-intel-core-i7-8565u-15-6-8gb-hd-1-tb-geforce-mx110-windows-10?_lc=11#analise-do-especialista) de R$ 4.299,00.
+
+|Produto|Quantidade|Valor unitário|Valor absoluto|
+|--|--|--|--|
+|Notebook|10|R$4.299,00|R$42.990,00|
+|Total|||R$42.990,00|
+
+## Ferramentas
+|Ferramentas|Descrição|Valor|
+|--|--|--|
+|ZenHub|Ferramenta de gerenciamento de projetos ágeis|R$0,00|
+|SonarCloud|Ferramenta de verificação de qualidade de código|R$0,00|
+|Draw.io|Ferramenta para elaboração de diagramas|R$0,00|
+|Git e GitHub|Ferramenta de versionamento de código|R$0,00|
+|Telegram e Discord|Ferramentas de comunicação do time|R$0,00|
+|Docker|Ferramenta para criação de administração de ambientes isolados|R$0,00|
+|Sistema operacional|Ferramenta de uso para a construção da aplicação|R$0,00|
+|Editor de texto para o código|Ferramenta de editor de texto|R$0,00|
+|Servidor|Serviço de computação em nuvem|R$0,00|
```

Comment:
Talvez essa parte mude no futuro. Não precisa mexer agora

Reply:
Você quer que remova, @devsalula ?

Reply:
Pode deixar do jeito que está. A gente muda quando for preciso